app/styles/molecules/button-dropdown.less

```diff
@@ -0,0 +1,62 @@
+.oss-button-dropdown {
+  position: relative;
+
+  &__trigger {
+    border: 1px solid @upf-primary-rock-blue;
```

Comment:
Maybe use new color pallet

Reply:
it doesn't much the existing buttons so it looks weird when placed next to existing buttons (checked w/ Victor)